python_gdeploy/wrapper/gdeploy_wrapper.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import uuid
+
+GDEPLOY_CONFIG_PATH = "/var/run/python_gdeploy/config_"
+
+
+def add_section(section):
+    return "[" + section + "]\n"
+
+
+def cook_gdeploy_config(recipe):
```

Comment:
It resembles INI style. But its very different from it. for example; it has few sections where there will be only keys (no key=value pair as in case of INI), sections where value is a list and not a single value etc. Similarly there are quite a few variations from INI.  So could not use the INI parser